Jenkinsfile

```diff
@@ -1,7 +1,7 @@
 // define a tag according to the Docker tag rules https://docs.docker.com/engine/reference/commandline/tag/
 // the hash sign (#) is problematic when using it in bash, instead of working around this problem, just replace all
 // punctuation with dash (-)
-def projectShortName = "${env.JOB_NAME}" - "/${env.BRANCH_NAME}"
+def projectShortName = "task_behavior_ros"
```

Comment:
Would it be clearer to people not familiar with Jenkins to call this the repo name? or at least put in a comment that these are the same.